packages/block-editor/src/components/colors/test/utils.js

```diff
@@ -1,118 +0,0 @@
-/**
```

Comment:
It doesn't look like this change is intentional.

Reply:
Thanks for catching that! I've readded the file from `master`